calculatorFrame.java

```diff
@@ -111,191 +104,192 @@ public void init() {
 		clearButtonPanel.add(clearAll);
 		content.add(clearButtonPanel,BorderLayout.NORTH);
 	}
-	
+
 	public void actionPerformed(ActionEvent e) {
-		
+
 		//One button
-		if(e.getSource() == one){
-			if (flag == false){
+		if(e.getSource() == one) {
+			if (flag == false) {
```

Comment:
A variable called “flag” is a code-smell since it doesn’t describe how the flag is set or used. What is this actually doing?